libs/e2e/src/lib/describer/modelAssessment/modelOverview/ensureAllModelOverviewFeatureCohortsViewElementsAfterSelectionArePresent.ts

```diff
@@ -55,17 +55,12 @@ function assertNumberOfChartRowsEqual(
   if (selectedFeatures > 1) {
     expectedNumberOfCohorts = featureCohortView?.multiFeatureCohorts;
   }
-  console.log(selectedFeatures);
-  console.log(expectedNumberOfCohorts);
   if (Array.isArray(expectedNumberOfCohorts)) {
     cy.get(getChartItems(chartIdentifier))
       .its("length")
       .should("be.gte", expectedNumberOfCohorts[0])
       .and("be.lte", expectedNumberOfCohorts[1]);
   } else {
-    cy.get(getChartItems(chartIdentifier)).should(
-      "have.length",
-      expectedNumberOfCohorts
```

Comment:
I think these come from a settings file for each dataset (singleFeatureCohorts or multiFeatureCohorts).  I think it might be better to just set these to the correct values for the vision notebooks or only do this type of very loose check for vision notebooks only, since we still want to validate this is consistent for tabular and text notebooks.

Reply:
@imatiach-msft these tests are unfortunately so flaky that different python versions were showing different values and hence failing at different steps. For diff versions, this gate ([Exif Nested Metadata support for Image Dashboards · microsoft/responsible-ai-toolbox@34f2f52 (github.com)](https://github.com/microsoft/responsible-ai-toolbox/actions/runs/8090180489/job/22107624226?pr=2542) has failed across all 3 notebooks.